dxw/dxw-media-controls
# Chore/migrate to php8.3

This PR upgrades the plugin for PHP 8.3 support.

- Updates the composer platform and dependencies
- Lints
- Adds a changelog

## How to test

1. `composer install & vendor/bin/psalm && vendor/bin/php-cs-fixer fix --dry-run` to run the linting
2. Add this branch of the plugin to a local site already migrated to PHP8 (e.g. Essex blogs), and activate the plugin
3. Under Settings > Additional media settings in the site, toggle "Show user guidance" to "Yes" and update the settings
4. You should see the guidance text (default: "It is recommended to use JPG or GIF filetypes for most images and to keep the file size as small as possible, for example between 100-200 KB. Use medium rather than large versions in posts") when uploading media.
